### PR TITLE
Change DG appendix C

### LINKED ISSUE #13: (3) Update the DG: user stories, glossary, NFRs, use cases

### PR DESCRIPTION
**Pull request by sp-shaopeng**
I don't know if it is the thing they want.

### CODE REVIEW

**Review by zwasd — APPROVED** [review]

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -326,41 +326,281 @@ _{More to be added}_
...
+* 2. Invalid ID.
++
+Use case back to step 1.
+
+[discrete]
+=== Use case: Repeat
+
+*MSS*
+
+1.  User requests to add an fix spending per month. Format: `Repeat -i INFO -a AMT -d DATE -p INTERVAL MONTH [-t TAG]`
```
Comment (EdmondOng) [inline docs/DeveloperGuide.adoc]: Format : 'repeat ...' 
Not consistent. Background not highlighted when viewed in html.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -326,41 +326,281 @@ _{More to be added}_
...
-=== Use case: Delete person
+=== Use case: Add
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to add a spending. Format: `Add -i INFO -a AMT [-t tag] [-d DATE] `
```
Comment (EdmondOng) [inline docs/DeveloperGuide.adoc:336]: Is this a 'Add -i ...' : format error?

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -326,41 +326,281 @@ _{More to be added}_
...
+3.  If tag is not specified during the instruction, `$AVE IT` will auto assign it to be `others`
++
+
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1. Wrong Format.
+* 2. Invalid ID.
```
Comment (EdmondOng) [inline docs/DeveloperGuide.adoc:366]: Should it be 1b. Invalid ID?

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -326,41 +326,281 @@ _{More to be added}_
...
+2.  `$AVE IT` will acknowledge.
++
+
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1. Wrong Format.
++
```
Comment (EdmondOng) [inline docs/DeveloperGuide.adoc:403]: Maybe you will want to include invalid ID?

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -326,41 +326,281 @@ _{More to be added}_
...
+* 1b. Invalid account name.
++
+Use case back to step 1.
+
+[discrete]
+=== Use case: Acc
+
+*MSS*
+
+1.  User requests to view all accounts. Format: `acc`.
```
Comment (EdmondOng) [inline docs/DeveloperGuide.adoc:514]: Should it be Acc?

**Review by EdmondOng — COMMENTED** [review]
Overall it's ok, do review my comments.

**Pull request merged**